security/pkg/nodeagent/sds/server.go

```diff
@@ -124,6 +124,9 @@ type Options struct {
 
 	// ClusterID is the cluster ID
 	ClusterID string
+
+	// Whether to generate EC private keys.
+	ECCKeys bool
```

Comment:
Same comment - this should be a string, if empty we default to RSA, otherwise use the curve.